### ISSUE TITLE
Can't start keybase

### ISSUE THREAD

**Issue by mlassi** [keybase/client#3986]
I installed keybase on OS X El Capitan 0.11.6 (15G31) via the dmg. When I try to start it via keybase ctl start I get the error:

I've tried 

my log id: 05ee27246a160e2b73e5901c

**Comment by maxtaco** [comment]
CC: @gabriel 

Looks like keybase the service isn't running.

**Comment by magnuslassislalom** [comment]
My original post got cut off. I tried starting it with keybase ctl start but the service fails to start up. I tried keybase -d install but it also failed starting the service. I installed it via the keybase.io dmg download. My user is an admin but I don't have permission to view anything inside the /keybase directory. I'm not sure if that could be related to the issue or different issue.

Thanks for the help.
